packages/utilities/tests/pickRandom.test.ts

```diff
@@ -0,0 +1,21 @@
+import { pickRandom } from '../src';
+
+describe('pickRandom', () => {
+	test('GIVEN array THEN picks one random element', () => {
+		const array = ['a', 'b', 'c'];
+		expect(array).toContain(pickRandom(array));
+	});
+
+	test('GIVEN count of one THEN picks one random element', () => {
+		const array = ['a', 'b', 'c'];
+		expect(array).toContain(pickRandom(array, 1));
+	});
+
+	test('GIVEN count of two THEN picks two random elements', () => {
+		const array = ['a', 'b', 'c'];
+		const picked = pickRandom(array, 2);
+		expect(array).toContain(picked[0]);
```

Comment:
You can use https://vitest.dev/api/#expect-arraycontaining too

Reply:
I think that has to be used in an equality check

Reply:
```ts
expects(array).toEquals(expects.arrayContaining(pickRandom(array, 2)));
```

Reply:
Oh, I see. Yeah sure